### ISSUE TITLE
Antergos-welcome includes but never used

### ISSUE THREAD

**Issue by Yoshi2889** [Antergos/Cnchi#730]
I assume it is used to provide a first start experience, but it is never used yet does get installed.

**Comment by karasu** [comment]
Yep, we never fully finished antergos-welcome, so that's why it's never run. As you can see, we have our hands plenty :(

**Comment by karasu** [comment]
https://github.com/Antergos/antergos-welcome

**Comment by stratus-ss** [comment]
Do you have a road map for what you are trying to achieve in the welcome? This may be something I can pick away at

**Comment by karasu** [comment]
The intention was to use pamac to install packages... I really don't remember at what point this was...

And to be honest, I don't like the pamac dependency. This should use pyalpm (but it will be more work, of course).

Maybe we could release this without the installing apps part and then think about it.

**Comment by lots0logs** [comment]
@karasu I hate to rain on everyone's parade but I am strongly against releasing this with the current design. I dont want to have ours look like Manjaro's which in turn looks like Koroa's which in turn looks like Ubuntu MATE's (for obvious reasons).

**Comment by karasu** [comment]
>rain on everyone's parade
First time I see this idiom, I had to look it up 😬 

I totally agree, to be fair. As you know, design is not one of my strong points, that's why I never spent time trying to redesign it. Maybe I'll give it a try.

**Comment by lots0logs** [comment]
Of course if I can find the time I'll pitch in, just not sure when I'll actually have time :grimacing:
